build-system/tasks/presubmit-checks.js

```diff
@@ -48,6 +48,9 @@ const realiasGetMode =
 // Terms that must not appear in our source files.
 const forbiddenTerms = {
   'DO NOT SUBMIT': '',
+  'whitelist': {
```

Comment:
Should this be `allowlist` and not `whitelist`?

Reply:
Never mind, I misunderstood the `forbiddenTerms` section. This is fine. Carry on!

Reply:
This is the forbidden term. `whitelist` is forbidden, and `allowlist` is suggested as the alternative.